website/views.py

```diff
@@ -12,6 +12,9 @@
 
 from alerts.models import Alert
 
+def oscnRedirect(request):
```

Comment:
The changes to this file is not needed to fix the issue.

Reply:
Done